docs/pages/machine-id/workload-identity.mdx

```diff
@@ -0,0 +1,136 @@
+---
+title: Workload Identity
+description: Teleport's Workload Identity for SPIFFE
+---
+
+<Admonition type="tip" title="Preview">
+Teleport Workload Identity is currently in Preview. We are actively working on
+improving the feature and would love to hear your feedback. We currently do
+not recommend using this feature in production.
```

Comment:
```suggestion
```
added in the above comment, we could also switch this out for a form https://www.surveymonkey.com/r/86ZQRZB